### ISSUE TITLE
Scaladoc for Scala3 shows assertion error

### ISSUE THREAD

**Issue by xerial** [wvlet/airframe#2430]
```
Problem parsing airframe-log/src/main/scala/wvlet/log/LogLevel.scala:[561..575..2255], documentation may not be generated.
java.lang.AssertionError: assertion failed

Problem parsing airframe-log/src/main/scala/wvlet/log/io/StopWatch.scala:[596..614..9397], documentation may not be generated.
java.lang.AssertionError: assertion failed
  | ... at scala.runtime.Scala3RunTime$.assertFailed(Scala3RunTime.scala:11)
  | => dat dotty.tools.dotc.util.Spans$Span$.start$extension(Spans.scala:45)
```

**Comment by exoego** [comment]
Minimized and reported in https://github.com/lampepfl/dotty/issues/16546

**Comment by xerial** [comment]
This issue has been fixed in Scala 3.3.0.
